### ISSUE TITLE
[v3] Error message: Material-UI: the key `adornedEnd` provided to the classes prop is not implemented in ForwardRef(Input). You can only override one of the following: root,formControl,focused,disabled,colorSecondary,underline,error,multiline,fullWidth,input,inputMarginDense,inputMultiline,inputTypeSearch.

### ISSUE THREAD

**Issue by Dina-Lai** [marmelab/react-admin#3987]
**What happened instead:**

I use a SelectInput in a SimpleForm mentioned in document (https://marmelab.com/react-admin/doc/3.0-beta/Inputs.html#selectinput).
It can work, but it will show the error message.

**Related code:**

Here is how I use a SelectInput in a SimpleForm

```
const choices = [
    { _id: 123, full_name: 'Leo Tolstoi', sex: 'M' },
    { _id: 456, full_name: 'Jane Austen', sex: 'F' },
];

export default (props) => (
    <Edit {...props}>
            <SimpleForm
                variant="standard"
                submitOnEnter={false}
                redirect={false}
            >
                <TextInput source="plan_name" resettable />
                <SelectInput source="author_id" choices={choices} optionText="full_name" optionValue="_id" />
            </SimpleForm>
    </Edit>
)
```
**Environment**

* React-admin version: 3.0.0-beta.6
* React version: 16.11.0
* Browser: Chrome
* Stack trace (in case of a JS error):

Material-UI: the key `adornedEnd` provided to the classes prop is not implemented in ForwardRef(Input).
You can only override one of the following: root,formControl,focused,disabled,colorSecondary,underline,error,multiline,fullWidth,input,inputMarginDense,inputMultiline,inputTypeSearch. 
    in WithStyles(ForwardRef(Input)) (created by Select)
    in Select (created by WithStyles(ForwardRef(Select)))
    in WithStyles(ForwardRef(Select)) (created by TextField)
    in TextField (created by WithStyles(ForwardRef(TextField)))
    in WithStyles(ForwardRef(TextField)) (created by ResettableTextField)
    in ResettableTextField (created by SelectInput)
    in SelectInput (at PlanEdit.js:42)
    in Anonymous (created by WithPermissions)
    in WithPermissions (created by Router.Consumer)
    in Router.Consumer (created by Route)
    in Route (created by ResourceRoutes)
    in ResourceRoutes (created by Resource)
    in Resource (at App.js:27)
    in App (at src/​index.js:7)

**Comment by fzaninotto** [comment]
Reproduced and confirmed, thanks for the report.

**Comment by fzaninotto** [comment]
Fixed by #3988
